content/_changelogs/11.0.0.md

```diff
@@ -1,17 +1,100 @@
+the breaking changes in more detail.**
+
+- Removed `cssFile`, `cssFiles`, `style`, `styles`, `stylesheet`, and
+  `stylesheets` from the `options` argument to the `cy.mount()` function. We
+  recommend writing test-specific styles in a separate `css` file you import in
+  your test, or in your `supportFile`. Addresses
+  [24328](https://github.com/cypress-io/cypress/issues/24328)
+- `mountHook` from `cypress/react` has been removed. We recommend replacing it
+  with `mount` and a component. Addresses
+  [24328](https://github.com/cypress-io/cypress/issues/24328)
+- `unmount` from `cypress/react` has been removed. We recommend using the API
+  React provides for unmounting components,
+  [unmountComponentAtNode](https://reactjs.org/docs/react-dom.html#unmountcomponentatnode).
+  Addresses [24328](https://github.com/cypress-io/cypress/issues/24328)
+- `mountCallback` from `cypress/vue` has been removed. We recommend using
+  `mount`. Addresses [24328](https://github.com/cypress-io/cypress/issues/24328)
+- When providing an inline `viteConfig` inside of `cypress.config`, any
+  `vite.config.js` file is not automatically merged. Addresses
+  [#24133](https://github.com/cypress-io/cypress/issues/24133)
+- Angular providers passed as part of the mounting options will be assigned at
+  the module level using the `TestBed.configureTestingModule` API. This means
+  that module-level providers (resolved from imports or
+  `@Injectable({ providedIn: 'root' })` can be overridden, but providers
+  specified in `@Component({ providers: [...] })` will not be overridden when
+  using `cy.mount(MyComponent, { providers: [...] })`. To override
+  component-level providers, use the `TestBed.overrideComponent` API. Addresses
+  [#24047](https://github.com/cypress-io/cypress/issues/24047) and
+  [#23427](https://github.com/cypress-io/cypress/issues/23427)
+
+**Experimental Breaking Changes:**
+
+- [`cy.session()`](/api/commands/session) now requires a setup command to use
+  the command. Addresses
+  [#24311](https://github.com/cypress-io/cypress/issues/24311).
 
 **Features:**
 
-**Bugfixes:**
+- Incorporated V8 snapshots into the build process of the Electron binary to
+  improve startup time and reduce the time to download and unzip the binary
+  during installation. Addresses
+  [#18480](https://github.com/cypress-io/cypress/issues/18480).
+- `cy.getCookie()`, `cy.getCookies()`, `cy.setCookie()`, `cy.clearCookie()`, and
+  `cy.clearCookies()` now accept a domain option. Addresses
+  [#24264](https://github.com/cypress-io/cypress/issues/24264)
+- Communication with the Cypress Dashboard will now verify CAs and reject any
+  unauthorized calls. If you use a self-signed CA you will need to set
+  `npm_config_ca`, `npm_config_cafile`, or `NODE_EXTRA_CA_CERTS`. Addresses
+  [#23980](https://github.com/cypress-io/cypress/issues/23980)
 
-**Dependencies:**
+**Bugfixes:**
 
-**Misc:**
+- Subsequent `cy.mount` calls within the same test will remove the last mounted
```

Comment:
This should be moved up to `Breaking changes`. Sorry if it wasn't marked properly in the commits! If a user was relying on the previous behavior where they could mount multiple components in a single test this change will break them.

Reply:
No, it was correctly marked, I just missed it. Moved up to breaking changes section.